geometry_msgs/Vector3[] Array